framework/core/Utils.pm

```diff
@@ -249,6 +249,15 @@ sub read_config_file {
 
 Check whether C<vid> represents a valid version id, i.e., matches \d+[bf].
 
+If C<vid> is not valid we terminate with an error message. Otherwise, we return
+a hash mapping C<{bid> to the bug-id parsed from C<vid> and C<type> to the type
+(C<b> or C<f>) of ther version.
```

Comment:
`This subroutine terminates with an error message, if C<vid> is not valid. Otherwise, it returns a hash that maps C<bid> to the bug id, parsed from the provided C<vid>, and C<type> to the version type (C<b> or C<f>), parsed from the provided C<vid>.`

Reply:
done